src/internal/operators/startWith.ts

```diff
@@ -1,7 +1,7 @@
-import { concat } from '../observable/concat';
-import { OperatorFunction, SchedulerLike, ValueFromArray } from '../types';
-import { popScheduler } from '../util/args';
+import { subscribeToArray } from '../observable/innerFrom';
+import { OperatorFunction, ValueFromArray } from '../types';
 import { operate } from '../util/lift';
+import { createOperatorSubscriber } from './OperatorSubscriber';
 
 // Devs are more likely to pass null or undefined than they are a scheduler
 // without accompanying values. To make things easier for (naughty) devs who
```

Comment:
After the scheduler was removed, these signatures became useless. Maybe we should remove them?

Reply:
Are signatures with `null` and `undefined` still relevant?

Reply:
I think they aren't relevant anymore.